### PR TITLE
docs: testnet update

### PR DESCRIPTION
**Pull request by nud3l**

### CODE REVIEW

File: docs/docs/build/getting-started/networks.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ BOB is fully EVM compatible so you can use any tools you are used to from Ethere
 - Description: Official public BOB testnet
 - Chain ID: 111
 - RPC URL (BOB): https://testnet.rpc.gobob.xyz/
-- Explorer (BOB): 
+- Explorer (BOB): https://explorerl2new-puff-bob-jznbxtoq7h.t.conduit.xyz/
```
Comment (nud3l) [inline docs/docs/build/getting-started/networks.md:17]: We need a dedicated URL for this.

File: docs/docs/build/contracts/index.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+## BOB Testnet (Sepolia)
+
+### Multicall3
+
+- Multicall3: `0x089b191d95417817389c8eD9075b51a38ca46DE8`
+
+### BOB Core Contracts
+
+- tBTC Relay: ``
```
Comment (nud3l) [inline docs/docs/build/contracts/index.md:11]: Please add this @sander2 @gregdhill

File: docs/docs/build/contracts/index.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+### Tokens (ERC20)
+
+- Erc20Minter (faucet): `0x7884560F14c62E0a83420F17832988cC1a775df1`
+- TBTC:
```
Comment (nud3l) [inline docs/docs/build/contracts/index.md:16]: Please deploy tBTC if we haven't done that yet

File: docs/docs/build/contracts/index.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+- USDC: `0x27c3321E40f039d10D5FF831F528C9CEAE601B1d`
+- USDT:
```
Comment (nud3l) [inline docs/docs/build/contracts/index.md:18]: Please also deploy USDT

File: docs/docs/build/contracts/index.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+## BOB Testnet (Sepolia)
+
+### Multicall3
+
+- Multicall3: `0x089b191d95417817389c8eD9075b51a38ca46DE8`
+
+### BOB Core Contracts
+
+- tBTC Relay: `0x077c5ed60fABb260784891786c6573373fDa8A3E`
```
Comment (sander2) [inline docs/docs/build/contracts/index.md:11]: @gregdhill can you confirm this is the correct contract that I deployed? https://github.com/bob-collective/bob/blob/mt-and-aa2/src/relay/DummyRelay.sol

Comment (gregdhill) [reply docs/docs/build/contracts/index.md:11]: We need to deploy the `TestLightRelay` on the master branch.

Comment (sander2) [reply docs/docs/build/contracts/index.md]: fixed

File: docs/docs/build/contracts/index.md (hunk 2 above)
Comment (tomjeatt) [reply docs/docs/build/contracts/index.md]: Done

File: docs/docs/build/contracts/index.md (hunk 3 above)
Comment (tomjeatt) [reply docs/docs/build/contracts/index.md]: Done

**Comment by nud3l** [comment]
This looks good to me. Anything else we should add?

**Comment by tomjeatt** [comment]
> This looks good to me. Anything else we should add?

No, I think that's it—I've been through the docs this morning and can't see anything else that needs updating. Only thing left to do is update the README files in the demos, to make sure the contract addresses, links etc. are all correct. Once I've done that I'll double check that everything matches up, and then we can merge this in.

**Review by tomjeatt — APPROVED** [review]

**Comment by tomjeatt** [comment]
Merging this in now—may be some small changes depending on how testing goes over the next day or two, but the bulk of the changes are done.

**Comment by nud3l** [comment]
Thanks!

**Pull request merged**